ios/RNTapResearch.m

```diff
@@ -97,38 +80,58 @@ - (dispatch_queue_t)methodQueue
   [placement showSurveyWallWithDelegate:self];
 }
 
-RCT_EXPORT_METHOD(setNavigationBarColor:(NSString *)hexColor)
+RCT_EXPORT_METHOD(displayEvent:(NSDictionary *)placementDict)
 {
-  UIColor *color = [RNTapResearch colorFromHexString:hexColor];
-  [TapResearch setNavigationBarColor:color];
+  if (!placementsCache) {
+    NSLog(@"Init placement wasn't called there is no available placement");
```

Comment:
since we don't perform "initPlacement(...)" anymore should this message be emended or removed?